### ISSUE TITLE
Server GC

### ISSUE THREAD

**Issue by sayedihashimi** [aspnet/Templates#575]
See https://github.com/aspnet/Docs/issues/1237

In the templates we have

``` json
"runtimeOptions": {
    "gcServer": true
}
```

It should be 

``` json
  "runtimeOptions": {
    "configProperties": {
      "System.GC.Server": true
    }
```

cc @danroth27

**Comment by danroth27** [comment]
@blackdwarf @piotrpMSFT Can you guys please confirm the server gc syntax above is correct?

**Comment by jcob-ross** [comment]
@danroth27

> All the aspnet repos already seem to use System.GC.Server
> https://github.com/aspnet/Tooling/issues/505#issuecomment-220504008

**Comment by phenning** [comment]
fixed with PR #577
